conf/haproxy-db.conf.example

```diff
@@ -42,11 +42,8 @@ frontend  main
 backend mysql
     mode tcp
     option tcplog
-    # disabled for now since it proved to be useless - jsemaan@inverse.ca
-    # TODO: remove it after cluster testing
-    #option external-check
-    #external-check command /usr/local/pf/var/run/db-check
-    #external-check path "/usr/bin:/bin"
+    #option httpchk
+    #default-server port 3307 inter 2s downinter 5s rise 3 fall 2 slowstart 60s maxconn 64 maxqueue 128 weight 100
```

Comment:
This should be either:
- Enabled by default (so uncomment the lines)
- Be configurable via the webadmin in the database_advanced section